src/app/services/http-service.service.ts

```diff
@@ -12,7 +12,28 @@ export class HttpServiceService {
     this.gitReleaseApiUrl = environment.gitApiUrl + "/releases";
   }
 
+  getPrDetails(){
+    const url = environment.githubApiUrl + "/repos/Worktez/worktez/pulls";
+    return this.httpClient.get(url);
+  }
+
   getReleaseDetails(){
     return this.httpClient.get(this.gitReleaseApiUrl);
   }
+
+  getGithubUserDetails() {
```

Comment:
Why are you entering your username here, and if this method is not used we can make good removal of it.

Reply:
done